cfy_manager/components/python/python.py

```diff
@@ -21,40 +21,43 @@
 from ...utils.install import yum_install, yum_remove
 from ...utils.files import copy_notice, remove_notice
 
-logger = get_logger(PYTHON)
-
-
-def _install():
-    if config[PYTHON]['install_python_compilers']:
-        logger.info('Installing Compilers...')
-        yum_install('python-devel', disable_all_repos=False)
-        yum_install('gcc', disable_all_repos=False)
-        yum_install('gcc-c++', disable_all_repos=False)
-
+from ..base_component import BaseComponent
 
-def _configure():
-    copy_notice(PYTHON)
-
-
-def install():
-    logger.notice('Installing Python dependencies...')
-    _install()
-    _configure()
-    logger.notice('Python dependencies successfully installed')
 
+logger = get_logger(PYTHON)
```

Comment:
Push to base class

Reply:
The reason it's in every sub class is because every component needs to log his own service name, thus the reason every get_logger(...) has a different service name.
Also, the service_names file where all the service names are brought from is not only used in the components classes but in outer modules as well...